NebraLtd/hm-pktfwd
# sx1302: upgrade hal to v2.1.0

**Why**


upgrade to v2.1.0 version of sx1302_hal so we are on the latest

**How**



upgrade hal to v2.1.0
- bring in latest repo from upstream
- update dockerfile
- update build script
- update sx1302 fixes (see #23) for updated repo
- remove some unecessary files and code
- update config.json files to add com_type and com_path and fine_timestamping

**References**


Closes: #24 
Closes: #42
Relates-to: #23
